client/app/components/ovh-angular-otrs/ovh-angular-otrs.less

```diff
@@ -1,11 +1,19 @@
 .otrs-popup {
+    height: 75% !important;
+    padding-top: 30px;
+    right: 30px;
+
     #headerPopup {
-        position: relative !important;
-        top: 0 !important;
+        position: absolute;
+        top: 0;
     }
 
-    #contentPopup {
-        height: auto !important;
+    #contentPopup { padding: 0 20px 20px 20px; }
+
+    .ovh-form-flat::after {
```

Comment:
Usage of class `.ovh-form-flat` isn't the best path to follow.
First you didn't have this dependency on Web, right?
Secondly this dependency is now deprecated.
@jleveugle Existing another option?

Reply:
Why ? It's the classname of the form inside `#contentPopup`. I could use `#otrsPopupForm`, but it's just to clear a float. Or put the clear on `#contentPopup`.